docs/UserGuide.adoc

```diff
@@ -284,13 +366,15 @@ e.g. `edit-s 2 n/James Lee e/jameslee@example.com` gp/apple 10
 * *Find supplier* : `find-s KEYWORD [MORE_KEYWORDS]` +
 e.g. `find-s James Jake`
 * *List supplier for a goods*: `source`
-* *Buy goods*: `buy n/SUPPLIER_NAME g/GOOD_NAME q/QUANTITY` +
-e.g. `buy n/Dave g/apple q/4`
-* *Sell goods*: `sell g/GOOD_NAME q/QUANTITY p/PRICE` +
-e.g. `sell g/apple q/4 p/5`
-* *Delete goods*: `delete-g INDEX` +
-e.g. `delete-g 2`
-* *Set minimum quantity for goods*: `warn INDEX q/MIN_QUANTITY`
+* *Buy goods*: `buy n/SUPPLIER_NAME g/GOOD_NAME q/QUANTITY x/EXPIRY_DATE d/TRANSACTION_DATE` +
+e.g. `buy n/Dave g/apple q/4 x/2003-08-01 d/2002-07-03`
+* *Sell goods*: `sell g/GOOD_NAME q/QUANTITY d/TRANSACTION_DATE` +
+e.g. `sell g/apple q/4 d/2030-01-05`
```

Comment:
Does not seem to match the buy and sell command described above, right?

Reply:
i think i will wait for jia da to correct it after he is done with new command format